daemon/c_service.h

```diff
@@ -112,6 +115,21 @@ c_service_start_child(c_service_t *service);
 int
 c_service_start_pre_exec(c_service_t *service);
 
+/**
+ * Send packed audit record to service.
+*/
```

Comment:
params and return values missing in description

Reply:
done